go.mod

```diff
@@ -1,21 +1,62 @@
 module github.com/ElrondNetwork/elrond-proxy-go
 
-go 1.13
+go 1.17
 
 require (
-	github.com/ElrondNetwork/elastic-indexer-go v1.0.7-0.20210625103944-3515df0722c5
-	github.com/ElrondNetwork/elrond-go v1.2.4-0.20210702131210-721bd99ea5bd
-	github.com/ElrondNetwork/elrond-go-logger v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v0.3.4-0.20210702112815-a12abaf3bf01
+	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
+	github.com/ElrondNetwork/elrond-go v1.3.27
+	github.com/ElrondNetwork/elrond-go-core v1.1.15
+	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
+	github.com/ElrondNetwork/elrond-go-logger v1.0.7
 	github.com/coinbase/rosetta-sdk-go v0.7.0
 	github.com/elastic/go-elasticsearch/v7 v7.12.0
 	github.com/gin-contrib/cors v0.0.0-20190301062745-f9e10995c85a
 	github.com/gin-contrib/pprof v1.3.0
-	github.com/gin-gonic/gin v1.7.2
+	github.com/gin-gonic/gin v1.7.6
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli v1.22.5
 	gopkg.in/go-playground/validator.v8 v8.18.2
 )
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.19 => github.com/ElrondNetwork/arwen-wasm-vm v1.3.20-0.20210702122719-c891907234fa
+require (
+	github.com/ElrondNetwork/concurrent-map v0.1.3 // indirect
```

Comment:
are these needed? if you remove them, run go mod tidy again, will they disappear?

Reply:
these were generated when I updated the go version